base/files/file_util_starboard.cc

```diff
@@ -345,7 +346,8 @@ bool CreateDirectoryAndGetError(const FilePath &full_path, File::Error* error) {
       continue;
     }
 
-    if (!SbDirectoryCreate(i->value().c_str())) {
+    if (mkdir(i->value().c_str(), 0700) != 0 &&
+      !SbDirectoryCanOpen(i->value().c_str())){
```

Comment:
Is this the right formatting? It looks like the wrong spacing for the conditional

Reply:
The linter won't automatically modify code under 3p libraries, like base, but we own this file entirely so we want to format it